main/Main.hs

```diff
@@ -9,6 +9,7 @@
 module Main where
 
 import Control.Concurrent (Chan, forkIO, newChan, readChan, writeChan)
+import Control.Exception (IOException, try)
```

Comment:
Please, only ever use `Control.Exception.Safe` from `safe-exceptions`.

Reply:
Also apparently it's in a wrong commit :roll_eyes: